tfhub_dev/assets/tulasiram58827/craft-text-detector/dr/1.md

```diff
@@ -0,0 +1,32 @@
+### Lite tulasiram58827/craft-text-detector/dr/1
+
+TF Lite quantized version of the CRAFT model for text detection from natural scenes and documents.
+
+<!-- parent-model: tulasiram58827/craft-text-detector/1 -->
+<!-- asset-path: https://github.com/tulasiram58827/craft_tflite/releases/download/v0.1.0/craft_model_dr.tar.xz -->
+
+[Open Colab Notebook](https://colab.research.google.com/github/tulasiram58827/craft_tflite/blob/main/colabs/CRAFT_TFLITE.ipynb).
+
+### About CRAFT
+
+The CRAFT model is a text detector that effectively detect text area by exploring each character region and affinity between characters. The bounding box of texts are obtained by simply finding minimum bounding rectangles on binary map after thresholding character region and affinity scores.
+
+- Model was quantized using `dynamic range` quantization as described [here](https://www.tensorflow.org/lite/performance/post_training_quant).
+
+- The Colab Notebook contains further information on running this model in real-time settings.
+
+- Original model weights are provided in this [repository](https://github.com/clovaai/CRAFT-pytorch)
+
+### Acknowledgments
+
+Thanks to SayakPaul and Khanh LeViet for constant guidance.
+
+### References
```

Comment:
Instead of using Bibtex format here, you can do something like - 

> Baek, Youngmin, et al. “Character Region Awareness for Text Detection.” ArXiv:1904.01941 [Cs], Apr. 2019. arXiv.org, http://arxiv.org/abs/1904.01941.

Reply:
Done.